staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -144,3 +144,78 @@ func (t *prefixTransformers) TransformToStorage(data []byte, context Context) ([
 	prefixedData = append(prefixedData, result...)
 	return prefixedData, nil
 }
+
+// LocationTransformer holds a transformer interface and the location of the resources it has to be used for.
+type LocationTransformer struct {
+	Location    []byte
```

Comment:
The location is always compared to context.AuthenticatedData(), which is `[]byte` in Clayton's code.

I personally don't see a case where the location shall not be a string, but had let it be `[]byte` for uniformity sake. The code bloat due to it seems more concerning, I'll change it to `string` then.